content/en/docs/ibm/create-cluster.md

```diff
@@ -128,12 +189,150 @@ ibmcloud ks cluster create ${WORKER_NODE_PROVIDER} \
 ```
 
+## Create an IBM Cloud Kubernetes cluster for `vpc-gen2` infrastructure
+
+Creating a `vpc-gen2` based cluster needs a VPC, a subnet and a public gateway attached to it. Fortunately, this is a one
+time setup. Future `vpc-gen2` clusters can reuse the same VPC/subnet(with attached public-gateway). 
+
+1. Begin with installing a vpc-infrastructure plugin:
+
+   `$ ibmcloud plugin install vpc-infrastructure`
+   
+   Refer to this [link](https://cloud.ibm.com/docs/containers?topic=containers-vpc_ks_tutorial), for more information.
+
+2. Target the gen 2 to access gen 2 resources:
+
+   `$ ibmcloud is target --gen 2`
+   
+   Verify, the target is correctly setup
+
+   ```shell
+   $ ibmcloud is target
+   Target Generation: 2
+   ```
+
+3. Create or use an existing VPC:
+   
+   a) Use an existing VPC: 
+   
+    ```shell
+    
+    $ ibmcloud is vpcs
+    Listing vpcs for generation 2 compute in all resource groups and region ...
+    ID                                          Name                Status      Classic access   Default network ACL                                    Default security group                                 Resource group   
+    r006-hidden-68cc-4d40-xxxx-4319fa3gxxxx   my-vpc1              available   false            husker-sloping-bee-resize                              blimp-hasty-unaware-overflow                           kubeflow   
+    
+    ```
+    If the above list contains the VPC that can be used to deploy your cluster - note it's ID.
+   
+   b) To create a new VPC, proceed as follows:
+
+    ```shell
+    $ ibmcloud is vpc-create my-vpc
+    Creating vpc my-vpc in resource group kubeflow under account IBM as ...
+                                                      
+    ID                                             r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx   
+    Name                                           my-vpc   
+    ...  
+    ```
+
+   **Save the ID in a variable `VPC_ID` as follows, so that we can use it later.**
+   
+   `export VPC_ID=r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx`
+
+4. Create or use an existing subnet:
+
+    a) To use an existing subnet:
+    
+    ```shell
+    $ ibmcloud is subnets
+    Listing subnets for generation 2 compute in all resource groups and region ...
+    ID                                          Name                      Status      Subnet CIDR       Addresses     ACL                                                    Public Gateway                             VPC                 Zone         Resource group   
+    0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   my-subnet                 available   10.240.128.0/18   16373/16384   husker-sloping-bee-resize                              my-gateway                                 my-vpc              us-south-3   kubeflow   
+    ```
+   If the above list contains the subnet corresponding to your VPC, that can be used to deploy your cluster - note it's
```

Comment:
```suggestion
    ```shell
    ibmcloud is subnets
    ```

    Example output:
    
    ```
    Listing subnets for generation 2 compute in all resource groups and region ...
    ID                                          Name                      Status      Subnet CIDR       Addresses     ACL                                                    Public Gateway                             VPC                 Zone         Resource group   
    0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   my-subnet                 available   10.240.128.0/18   16373/16384   husker-sloping-bee-resize                              my-gateway                                 my-vpc              us-south-3   kubeflow   
    ```

    If the above list contains the subnet corresponding to your VPC, that can be used to deploy your cluster - make sure you note it's
```